docs/proposals/scheduling/20230707-eviction-arbitrate-mechanism-in-descheduler.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Eviction Arbitration Mechanism in Descheduler
+authors:
+  - "@baowj-678"
+reviewers:
+  - @hormes
+  - @eahydra
+  - @FillZpp
+  - @ZiMengSheng
+  - @zwzhang0107
+  - @saintube
+  - @jasonliu747
+creation-date: 2023-07-07
+last-updated: 2023-07-21
+status: provisional
+---
+
+# Eviction Arbitration Mechanism in Descheduler
+
+## Table of Contents
+<!-- TOC -->
+
+- [Eviction-Arbitration-Mechanism-in-Descheduler](#eviction-arbitration-mechanism-in-descheduler)
+    - [Table of Contents](#table-of-contents)
+    - [Glossary](#glossary)
+    - [Summary](#summary)
+    - [Motivation](#motivation)
+        - [Goals](#goals)
+        - [Non-Goals/Future Work](#non-goalsfuture-work)
+    - [Proposal](#proposal)
+        - [User Stories](#user-stories)
+            - [Story 1](#story-1)
+            - [Story 2](#story-2)
+            - [Story 3](#story-3)
+        - [Pod Migration Job CRD Field](#pod-migration-job-crd-field)
+            - [Migration Job Spec](#migration-job-spec)
+        - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+            - [PodMigrationJob Controller](#podmigrationjob-controller)
+                - [Controller Process](#controller-process)
+                    - [Controller Reconcile Process](#controller-reconcile-process)
+                    - [Arbitration Process](#arbitration-process)
+                    - [Migration Process](#migration-process)
+                - [Arbitration Mechanism](#arbitration-mechanism)
+                    - [Sort PodMigrationJob](#sort-podmigrationjob)
+                    - [GroupFilter PodMigrationJob](#groupfilter-podmigrationjob)
+                    - [Select PodMigrationJob](#select-podmigrationjob)
+            - [Controller Configuration](#controller-configuration)
+    - [Alternatives](#alternatives)
+    - [Implementation History](#implementation-history)
+
+<!-- /TOC -->
+## Glossary
+
+## Summary
+
+This proposal designed a mechanism in descheduler to arbitrate `PodMigrationJob`, through which the system's stability could be improved when a mount of Pods or some important Pods is being evicted. 
+
+## Motivation
+
+Arbitrate Mechanism is an important capability that Pod Migration relies on, and Pod Migration is relied on by many components (such as deschedulers). But Pod Migration is a complex process,  involving steps such as auditing, resource allocation, and application startup, and is mixed with application upgrading, scaling scenarios, and resource operation and maintenance operations by cluster administrators. 
+
+So when a large number of Pods are simultaneously migrated, this may have some impact on the stability of the system. In addition, if many Pods of the same workload are migrated simultaneously, it will also have an impact on the stability of the application. Moreover, if a job's Pod migration takes too long, it can affect the job's completion time.
```

Comment:
It is more appropriate to use `concurrently` instead of `simultaneously`.

Reply:
OK